spec/controllers/idv/cancellations_controller_spec.rb

```diff
@@ -103,9 +103,18 @@
   end
 
   describe '#update' do
+    let(:enrollment) do
+      create(
+        :in_person_enrollment,
+        enrollment_code: '123',
+        id: '1',
+      )
+    end
+
     before do
       stub_sign_in
       stub_analytics
+      allow(controller).to receive(:enrollment).and_return(enrollment)
```

Comment:
Instead of mocking like this, better to do something like
```ruby
    let(:user) { create(:user) }
    let(:enrollment) { create(:in_person_enrollment, :pending, user: user) }

    before do
      stub_sign_in(user)
      stub_analytics
    end
```